Code/Source/solver/read_files.cpp

```diff
@@ -2243,35 +2243,37 @@ void read_outputs(Simulation* simulation, EquationParameters* eq_params, eqType&
   // 
   for (int j = 0; j < 3; j++) {
     for (int k = 0; k < nDOP[j+1]; k++) {
-      lEq.output[k].wtn[j] = true;
+      lEq.output[k].options.boundary_integral = false;
```

Comment:
@ktbolt is the loop on j still necessary here?

Reply:
@dcodoni It was not clear to me from the code why all of the `wtn[j]` were set to true so I left the loop. But the `boundary_integral` are all initialized to false anyway so I will remove the loop.